src/resources/scripts/toCoco.py

```diff
@@ -0,0 +1,116 @@
+import pandas as pd
+import numpy
+from shapely.geometry import Point
+from shapely.affinity import scale,rotate
+import warnings
+import sys
+import math
+import json
+import ast
+
+# target_file = sys.argv[2]
+# csv = sys.argv[3]
+
+# the path to the CSV file
+paths = sys.argv[3:]
+# json (output) path #ignore for now
+save_json_path = sys.argv[2]
+
+
+def check_shape(row):
+    # only polygons are accepted
+    shape = row.shape_name
+    flag_shape = 1
+    if not(shape == "LineString" or shape == "Polygon" or shape == "Rectangle" or shape=="Circle" or shape=="Ellipse") :
+        warnings.warn('The shape %s is not supported !' %(shape))
+        flag_shape = 0
+    return flag_shape
+
+def image(row):
+    image = {}
+    # row.attributes is a string, we want to transform it in a dict
+    desired_dict = json.loads(row.attributes)
```

Comment:
What happens if the cell is empty (see #87)?

Reply:
I used the script from community-scripts which I thought would be checked and valid. If row.attributes is empty the whole thing fails. Unfortunately the only alternative is to read the width and height from the file itself (which might not be available to the script ???).